arm-sql/2014-04-01/swagger/sql.core.json

```diff
@@ -2330,6 +2330,27 @@
           "format": "date-time",
           "description": "The recovery period start date of the Azure SQL Database. This records the start date and time when recovery is available for this Azure SQL Database (ISO8601 format)."
         },
+        "createMode": {
+          "type": "string",
+          "description": "Specifies the type of database to create.",
+          "enum": [
+            "Copy",
+            "Default",
+            "NonReadableSecondary",
+            "OnlineSecondary",
+            "PointInTimeRestore",
+            "Recovery",
+            "Restore"
+          ],
+          "x-ms-enum": {
+            "name": "createMode",
+            "modelAsString": true
+          }
+        },
+        "sourceDatabaseId": {
+          "type": "string",
+          "description": "Conditional.  Specifies the resource Id of the source database.  If createMode is not set to Default, then this value must be specified.  The name of the source database must be the same."
+        },
```

Comment:
Do I need to specify Collation, ServiceObjective, edition, macSizeByte when this database is going to be replicated, also what will happen in case I need to specify it whether this information needs to be same as source Database or it can be different from source database. Can sourceDatabaseId be changed once database is created.